sdk/common/src/test/java/io/opentelemetry/sdk/internal/PrimitiveLongListTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class PrimitiveLongListTest {
```

Comment:
I think it would be nice to add an assertion of sameness:
```
    long[] arr = new long[] {1, 2, 3};
    assertThat(PrimitiveLongList.toArray(PrimitiveLongList.wrap(arr))).isSameAs(arr);
```